SERVICE_DEPENDENCIES.md

```
@@ -0,0 +1,780 @@
+   - Data Stream Manager for Kafka
+   - Datadog tracing integration
+
+5. **rig (Next.js)**
+   - Producer: `kafkajs@2.2.4` dependency
+   - Event emission for system events
+
+**Kafka Topics:**
+
+| Topic | Producer | Consumer | Purpose |
+|-------|----------|----------|---------|
+| `tundra-beads-created` | td, rig | airflow, td | New task/bead creation events |
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)